apps/plato_single_variant/Readme.md

```diff
@@ -1,20 +1,17 @@
+This app performs genetic associations using PLATO (the PLatform for the Analysis, Translation and Organization of large-scale data). PLATO is a software tool equipped to handle whole-genome sequence data for hundreds of thousands of samples to explore complexity and genetic architecture underlying common traits using phenome-wide association studies (PheWAS), Genome-wide association studies (GWAS), genetic interactions, environment-wide association studies (EWAS); gene-environment interactions (GxE), and copy number and rare variant analyses.
 
-App performs genetic associations using PLATO (the PLatform for the Analysis, Translation and Organization of large-scale data). PLATO is a software tool equipped to handle whole-genome and sequence data for hundreds of thousands of samples to explore complexity and genetic architecture underlying common traits using phenome-wide association studies (PheWAS), Genome-wide association studies (GWAS), genetic interactions, environment-wide association studies (EWAS) and gene-environment interactions (GxE), and copy number and rare variant analyses
-
-Although, PLATO is platform to perform different kind of analysis, this app specifically focus on association analysis for single outcome (GWAS) or across multiple outcome (PheWAS).
+Although PLATO is a platform to perform different kinds of analyses, this app specifically focuses on single outcome (GWAS) and multiple outcome (PheWAS) analyses.
 
 ## Where can I find PLATO software manual?
-You can find the detailed PLATO software manual here: http://ritchielab.psu.edu/software/plato-download
+You can find the detailed PLATO software manual here: https://ritchielab.org/software/plato-download
 
 ## What data are required for this app to run?
 There are two input options required to run the app:
-- **Genotype Files**: It require input genotype files in binary PLINK format (`.bed`,`.bim`,`.fam`). Please refer to PLINK documentation for information on file format.
-- **Phenotype File**: An input phenotype file should be a tab-delimited, where first row indicates column headers and each subsequent row representing values for an independent sample. The first two columns should be Family ID (FID) and the Individual ID (IID) for the sample as in PLINK `.FAM` file, and rest of the columns should contain the numeric/categorical value for each phenotype. For example:
+- **Genotype Files**: It is required to input genotype files in binary PLINK format (`.bed`,`.bim`,`.fam`). Please refer to the [PLINK documentation](http://zzz.bwh.harvard.edu/plink/) for information on file format.
+- **Phenotype File**: An input phenotype file must be a tab-delimited, where first row indicates column headers and each subsequent row represents values for an independent sample. The first two columns must be the Family ID (FID) and Individual ID (IID) for the sample as in PLINK `.fam` file, and subsequent columns contain the numeric/categorical values for each phenotype.
```

Comment:
The input phenotype file must be tab-delimited, where the first row indicates column headers and each subsequent row represents values for an independent sample. The first two columns must be the Family ID (FID) and Individual ID (IID) for the sample as in PLINK `.fam` file and subsequent columns contain the numeric/categorical values for each phenotype.